core-web/libs/sdk/uve/src/lib/dom/dom.utils.ts

```diff
@@ -283,6 +284,27 @@ export function getDotContentletAttributes(
     };
 }
 
+/**
+ * Helper function that returns an object containing analytics-specific data attributes.
+ * These attributes are used by the DotCMS Analytics SDK to track content interactions.
+ *
+ * @param {DotCMSBasicContentlet} contentlet - The contentlet to get the analytics attributes for
+ * @returns {DotAnalyticsAttributes} The analytics data attributes
+ * @internal
+ */
+export function getDotAnalyticsAttributes(
+    contentlet: DotCMSBasicContentlet
+): DotAnalyticsAttributes {
+    return {
+        class: 'dotcms-analytics-contentlet',
```

Comment:
do we need a class or a another attr? 

You can also query by attributes

Reply:
Class-based queries are 3-5x faster than attribute-based queries when finding multiple elements:
```
// Fast path (~1-3ms for 1000 contentlets)
const allContentlets = document.querySelectorAll('.dotcms-analytics-contentlet');

// Slower path (~5-15ms for same query)
const allContentlets = document.querySelectorAll('[data-dot-analytics-identifier]');
```

Modern browser engines maintain optimized indexes for CSS classes
Data attributes, on the other hand, not use indexes